diff --git a/lib/api/Extractor.js b/lib/api/Extractor.js
index 781b5b59330c5a54feb01d06efe431aadd733b32..20cbdea06d359c8ba6f7379b2b992b43ed0a05b7 100644
--- a/lib/api/Extractor.js
+++ b/lib/api/Extractor.js
@@ -258,7 +258,7 @@ class Extractor {
         });
     }
     static _checkCompilerCompatibility(extractorConfig, messageRouter) {
-        messageRouter.logInfo("console-preamble" /* ConsoleMessageId.Preamble */, `Analysis will use the bundled TypeScript version ${ts.version}`);
+        // messageRouter.logInfo("console-preamble" /* ConsoleMessageId.Preamble */, `Analysis will use the bundled TypeScript version ${ts.version}`);
         try {
             const typescriptPath = resolve.sync('typescript', {
                 basedir: extractorConfig.projectFolder,
@@ -273,8 +273,8 @@ class Extractor {
                 const theirMajor = semver.major(packageJson.version);
                 const theirMinor = semver.minor(packageJson.version);
                 if (theirMajor > ourMajor || (theirMajor === ourMajor && theirMinor > ourMinor)) {
-                    messageRouter.logInfo("console-compiler-version-notice" /* ConsoleMessageId.CompilerVersionNotice */, `*** The target project appears to use TypeScript ${packageJson.version} which is newer than the` +
-                        ` bundled compiler engine; consider upgrading API Extractor.`);
+                    // messageRouter.logInfo("console-compiler-version-notice" /* ConsoleMessageId.CompilerVersionNotice */, `*** The target project appears to use TypeScript ${packageJson.version} which is newer than the` +
+                    //     ` bundled compiler engine; consider upgrading API Extractor.`);
                 }
             }
         }