diff --git a/lib/api/Extractor.js b/lib/api/Extractor.js
index d125dff6b2741441e2f38c43704abd86b1209e8d..3cfbd58755d910dc2181eacc235adf148ce7e088 100644
--- a/lib/api/Extractor.js
+++ b/lib/api/Extractor.js
@@ -281,7 +281,7 @@ class Extractor {
         return apiReportChanged;
     }
     static _checkCompilerCompatibility(extractorConfig, messageRouter) {
-        messageRouter.logInfo(ConsoleMessageId_1.ConsoleMessageId.Preamble, `Analysis will use the bundled TypeScript version ${ts.version}`);
+        // messageRouter.logInfo(ConsoleMessageId_1.ConsoleMessageId.Preamble, `Analysis will use the bundled TypeScript version ${ts.version}`);
         try {
             const typescriptPath = resolve.sync('typescript', {
                 basedir: extractorConfig.projectFolder,
@@ -296,8 +296,8 @@ class Extractor {
                 const theirMajor = semver.major(packageJson.version);
                 const theirMinor = semver.minor(packageJson.version);
                 if (theirMajor > ourMajor || (theirMajor === ourMajor && theirMinor > ourMinor)) {
-                    messageRouter.logInfo(ConsoleMessageId_1.ConsoleMessageId.CompilerVersionNotice, `*** The target project appears to use TypeScript ${packageJson.version} which is newer than the` +
-                        ` bundled compiler engine; consider upgrading API Extractor.`);
+                    // messageRouter.logInfo(ConsoleMessageId_1.ConsoleMessageId.CompilerVersionNotice, `*** The target project appears to use TypeScript ${packageJson.version} which is newer than the` +
+                    //     ` bundled compiler engine; consider upgrading API Extractor.`);
                 }
             }
         }
